tui/installer/__init__.py

```diff
@@ -141,6 +148,8 @@ def has_guest_disks_fn(answers):
              predicates=[lambda _:len(results['upgradeable-products']) > 0 or len(results['backups']) > 0]),
         Step(uis.upgrade_settings_warning,
              predicates=[upgrade_but_no_settings_predicate]),
+        Step(uis.migrate_to_corosync3,
+             predicates=[is_reinstall_fn, preserve_settings, corosync2_was_in_use]),
```

Comment:
This sounds like a step doing the migration, `migrate_to_corosync3_warning` would be easier to read